### PR TITLE
Dedup browsers

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/airtap/airtap/issues/168

### CODE REVIEW

**Comment by vweevers** [comment]
LGTM. Are the build failures related?

**Comment by ralphtheninja** [comment]
> LGTM. Are the build failures related?

Looking into it. This PR shouldn't cause any problems but I might have missed something of course.

**Comment by ralphtheninja** [comment]
Aaah of course, here it is:

https://github.com/airtap/airtap/blob/cb167a2ab1a2a35995516ba5cb38d240930df583/test/integration/tape-sauce.js#L41-L44

File: test/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require('./unit/aggregate-browsers')
-require('./unit/airtap')
+require('./unit/cli')
 require('./unit/browserify-builder')
-require('./unit/capabilities')
+require('./unit/airtap')
```
Comment (ralphtheninja) [inline test/index.js]: Just noticed now that this became out of order.

**Pull request merged**